### PR TITLE
Related Links & Chapters

### LINKED ISSUE #320: [Guides]  Could the Related Links on Guides pages populate down to the Chapters (subpages)
Experience Fragments can be created using the Franklin template, which can be added further in the page via code

### PR DESCRIPTION
**Pull request by amlan18041996**
Fix #320 

Test URLs:
- Before: https://main--danaher-abcam--aemsites.aem.page/en-us/technical-resources/guides/antibody-basics
- After: https://320-related-links-subpages--danaher-abcam--aemsites.aem.page/en-us/technical-resources/guides/antibody-basics
After: https://320-related-links-subpages--danaher-abcam--aemsites.aem.page/en-us/technical-resources/guides/antibody-basics/antibody-basics

### CODE REVIEW

File: eds/blocks/sidelinks/sidelinks.js [hunk 1]
```diff
@@ -32,13 +32,14 @@ export default function decorate(block) {
...
   const linkHeading = getMetadata('template').includes('guide') ? 'Related Links' : 'Explore our products';
   if (allParagraphs.length > 0) {
-    divEl.append(p({ class: 'text-sm leading-6 font-semibold uppercase text-[#65797C] px-2' }, linkHeading));
+    divEl.prepend(p({ class: 'text-sm leading-6 font-semibold uppercase text-[#65797C] px-3 py-2' }, linkHeading));
     allParagraphs.forEach((elPara) => {
-      applyClasses(elPara, 'block text-sm leading-6 font-semibold text-[#378189] p-2 border-b border-b-[#D8D8D8] hover:underline');
-      divEl.append(p({ class: 'leading-5 text-base font-medium text-[#378189] my-0' }, elPara));
+      applyClasses(elPara?.parentElement, 'leading-5 text-base font-medium text-[#378189] my-0');
+      applyClasses(elPara, 'block text-sm leading-6 font-semibold text-[#378189] px-3 py-2 border-b border-b-[#D8D8D8] hover:underline');
+      // divEl.append(p({ class: 'leading-5 text-base font-medium text-[#378189] my-0' }, elPara));
```
Comment (rgravitvl) [inline eds/blocks/sidelinks/sidelinks.js:42]: remove the commented line

File: eds/scripts/scripts.js [hunk 2]
```diff
@@ -911,6 +909,10 @@ async function loadPage() {
   loadDelayed();
 }
 
+export function createStickyBottom(element) {
```
Comment (rgravitvl) [inline eds/scripts/scripts.js:912]: why do we need this function in scripts??

**Review by rgravitvl — CHANGES_REQUESTED** [review]

File: eds/scripts/scripts.js (hunk 2 above)
Comment (amlan18041996) [reply eds/scripts/scripts.js:912]: We need this functionality as chapters block will not compile it out of chapters.css which we are expecting below footer( outside chapters block )

**Comment by davenichols-DHLS** [comment]
@rgravitvl , @sharanagoudapatil  and @amlan18041996  - please let me know if we need to discuss this?

**Review by rgravitvl — APPROVED** [review]

**Review by sharanagoudapatil — APPROVED** [review]

**Pull request merged**